tests/e2e/specs/empty-trash-posts.test.js

```diff
@@ -0,0 +1,43 @@
+import {
+    visitAdminPage,
+    createNewPost,
+    trashAllPosts,
+    publishPost
+} from '@wordpress/e2e-test-utils';
+
+describe( 'Empty Trash', () => {
+    beforeEach( async () => {
+        await trashAllPosts();
+    } );
+
+    it( 'displays a message in the posts table when no posts are present', async () => {
+        await visitAdminPage( '/edit.php' );
+        const noPostsMessage = await page.$x(
+            '//td[text()="No posts found."]'
+        );
+        expect( noPostsMessage.length ).toBe( 1 );
+    } );
+
+    it( 'Empty Trash', async () => {
+
+        //Create a Post
+        const title = 'Test Title';
+        await createNewPost( { title } );
+        await publishPost();
+
+        await visitAdminPage( '/edit.php' );  
+
+        // Move post to trash
+        await page.waitForSelector( '#the-list .type-post' );
+        await page.hover('.row-title');
+        await page.click("a[aria-label='Move “Test Title” to the Trash']");
+
+        // Empty trash 
+        await page.waitForSelector(".subsubsub");
+        await page.click("a[href='edit.php?post_status=trash&post_type=post']");
+        await page.waitForSelector( '#delete_all' );
+        await page.click("#delete_all");
+		await page.waitForSelector("#message");
```

Comment:
Yes I have specified the reason [here](https://github.com/WordPress/wordpress-develop/pull/2186#issuecomment-1041162514)

Reply:
> the number of posts were getting deleting dynamically.

I think I'm missing something here, but why would the number of posts be dynamic? Is it because we're not clearing up states between tests? Or are we parallelizing tests that causes race-condition?